### PR TITLE
Add scramble secrecy requirements to the Regulations

### PR DESCRIPTION
**Pull request by CarterKoala**

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -391,11 +390,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A1b2) If a time from the Stackmat timer is available, it is the original recorded time. Otherwise (e.g. the Stackmat timer malfunctioned, ran for over 10 minutes or was not used at all), the stopwatch time is the original recorded time.
     - A1c) A competitor participating in an event must be able to fulfill the event's requirements (see [Regulation 2k6](regulations:regulation:2k6)).
 - A2) Scrambling:
-    - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. The competitor then waits in their assigned Competitor Waiting Area until they are called to compete.
+    - A2a) When called for a round, the competitor submits a solved puzzle to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
    - A2a) When called for a round, the competitor submits a solved puzzle to the scrambler.
```
I guess this part is not needed with A2a1 and A2a2 below? It's too repetitive

File: wca-regulations.md [hunk 2]
```diff
@@ -80,6 +79,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 2f) Competitors must obey venue rules and conduct themselves in a considerate manner.
 - 2g) Competitors must remain quiet when inside the designated competition area. Talking is permitted, but must be kept at a reasonable level, and away from competitors who are actively competing.
     - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all such competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
+        - 2g3a) For rounds where results are being entered live, the organization team may enforce that competitors are not permitted to view the results entered from their group until they have completed their attempts.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - 2g3a) For rounds where results are being entered live, the organization team may enforce that competitors are not permitted to view the results entered from their group until they have completed their attempts.
```

File: wca-regulations.md [hunk 3]
```diff
@@ -391,11 +390,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
 - A2) Scrambling:
-    - A2a) When called for a round, the competitor submits the puzzle, in its solved state, to the scrambler. The competitor then waits in their assigned Competitor Waiting Area until they are called to compete.
+    - A2a) When called for a round, the competitor submits a solved puzzle to the scrambler. If a Competitor Waiting Area is in use, the competitor then waits in their assigned Competitor Waiting Area until they are called to compete. If a Competitor Waiting Area in not in use, the competitor then waits at their assigned solving station.
+        - A2a1) If a Competitor Waiting Area is in use, the competitor must remain within their assigned Competitor Waiting Area until they are called to complete an attempt. The competitor must return to their assigned Competitor Waiting Area after completing each attempt if they have more attempts in the round. Exception: competitors may leave the Competitor Waiting Area with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - A2a1) If a Competitor Waiting Area is in use, the competitor must remain within their assigned Competitor Waiting Area until they are called to complete an attempt. The competitor must return to their assigned Competitor Waiting Area after completing each attempt if they have more attempts in the round. Exception: competitors may leave the Competitor Waiting Area with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate.
```

File: wca-guidelines.md [hunk 4]
```diff
@@ -179,7 +179,9 @@ To be more informative, each Guideline is classified using one of the following
...
 - A1a3+) [REMINDER] The organization team and the WCA Delegate must be mindful that time limits influence the strategies of the competitors (e.g. rushing the first two attempts in hopes of meeting a cutoff in a cutoff round), and that changing time limits after the start of the round can disadvantage some competitors unfairly.
 - A1a4+) [REMINDER] If a competitor has accidentally been permitted to exceed the time limit, the time limit must be enforced retroactively, and the judge, competitor, and WCA Delegate should be informed (see [Regulation 1g2](regulations:regulation:1g2)). Judges must always be aware of the time limit for a current attempt (which might depend on previous attempts, in the case of a cumulative time limit).
+- A2a3+) [REMINDER] Competitors should not be permitted to view the scramble sequences for their own group even after having completed all their attempts, unless permission has been granted by the WCA Delegate (see [Guideline 4b2+](guidelines:guideline:4b2+)).
```
Comment (CarterKoala) [inline wca-guidelines.md]:
```suggestion
- A2a3+) [REMINDER] Competitors should not be permitted to view the scramble sequences for their own group even after having completed all their attempts, unless permission has been granted by the WCA Delegate (see [Guideline 4b2+](guidelines:guideline:4b2+)).
```

File: wca-regulations.md (hunk 1 above)
Comment (ohexter) [reply wca-regulations.md]: Agreed

File: wca-regulations.md [hunk 5]
```diff
@@ -391,11 +390,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
+        - A2a2) If a Competitor Waiting Area is not in use, the competitor must remain at their assigned solving station until they have completed all their attempts in the round. Exception: competitors may leave their assigned solving station with permission from the WCA Delegate. Penalty: disqualification of the remaining attempts (DNF), at the discretion of the WCA Delegate.
+        - A2a3) Once called for a round, a competitor must not attempt to enter any area assigned for scrambling until all competitors in their group have no further attempts, unless permission has been granted by the WCA Delegate.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
        - A2a3) Once called for a round, a competitor must not attempt to enter any area assigned for scrambling until all competitors in their group have no further attempts, unless permission has been granted by the WCA Delegate.
```

File: wca-regulations.md [hunk 6]
```diff
@@ -42,7 +42,6 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 1a) A competition must include a WCA Delegate and an organization team (consisting of one or more individuals) with the following officials: judges, scramblers and score takers.
 - 1b) The organization team of a competition is responsible for logistics before, during, and after the competition.
 - 1c) The WCA Delegate is responsible for ensuring that the competition adheres to the WCA Regulations and any applicable WCA policies or requirements. The WCA Delegate may appoint other members of the organization team to carry out specific responsibilities on their behalf, but is ultimately accountable for how these responsibilities are carried out.
-    - 1c1) The WCA Delegate must ensure that the competition follows the [WCA Scramble Accountability Policy](https://documents.worldcubeassociation.org/documents/policies/external/Scramble%20Accountability.pdf).
```
Comment (mintydev789) [inline wca-regulations.md:45]: Why has this been removed? The WSAP has plenty of points not included in this update.

Comment (CarterKoala) [reply wca-regulations.md:45]: 1c Already covers this - there isnt a need to specifically call out WSAP and not WCRP for example

Comment (mintydev789) [reply wca-regulations.md:45]: Oh, right, makes sense

**Comment by mintydev789** [comment]
Just a small note, I think the commit titles should be more descriptive, because the history is kinda hard to follow in this PR

File: wca-regulations.md [hunk 7]
```diff
@@ -391,11 +390,15 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
     - A2b) A scrambler scrambles the puzzle according to the regulations in [Article 4](regulations:article:4).
         - A2b1) For Square-1, the organization team may enforce placing a thin object in the puzzle to prevent accidental moves from being applied before the start of the attempt. If these objects are used, the organization team should make an announcement before the round starts.
     - A2c) After the scrambler starts scrambling the puzzle, the competitor must not see the puzzle until the inspection phase starts.
         - A2c1) The scrambler places a cover over the scrambled puzzle that makes it impossible for any competitors or spectators to see any part of the puzzle. The cover remains over the puzzle until the start of the attempt.
+        - A2c2) When competing, competitors must not actively try to watch another competitor complete an official attempt. If the competitor watches another competitor attempt a scramble they have not yet received, the corresponding attempt is recorded as DNF.
```
Comment (ohexter) [inline wca-regulations.md:406]:
```suggestion
        - A2c2) When competing, competitors must not actively try to watch another competitor complete an official attempt. If the competitor watches another competitor attempt a scramble they have not yet received, the corresponding attempt is recorded as DNF. Exception: Competitors in 3x3x3 Multi-Blind who have completed or declined an attempt may watch other competitors complete that attempt.
```
In response to feedback. Not 100% sure this is needed as it's not clear whether those competitors would fit the "while competing" criterion.

Comment (CarterKoala) [reply wca-regulations.md:406]: Closing as this is covered by "When competing"

**Review by ohexter — APPROVED** [review]
